benches/plane.rs

```diff
@@ -19,16 +19,23 @@ pub fn downsample_8bit(c: &mut Criterion) {
   let input = init_plane_u8(1920, 1080);
   c.bench_function("downsample_8bit", move |b| {
     b.iter(|| {
-      let _ = input.downsampled(960, 540);
+      let _ = input.downsampled(input.cfg.width, input.cfg.height);
     })
   });
 }
 
 pub fn downsample_odd(c: &mut Criterion) {
   let input = init_plane_u8(1919, 1079);
+  let mut padded = Plane::new(input.cfg.width, input.cfg.height, 0, 0, 1, 1);
```

Comment:
I think it would make more sense for this padding code to be in init_plane_u8(), if we decide that all planes must be padded like this.

Reply:
Thanks. I have moved this part to init_plane_u8().

Reply:
Fixed.